arches_for_science/templates/views/components/plugins/init-workflow.htm

```diff
@@ -1,13 +1,97 @@
-                <span><i data-bind="attr:{class: ('fa '+workflow.icon +' workflow-select-wf-icon')}"></i></span>
+{% load i18n %} 
+
+<div>
+    <!-- Modal -->
+    <!-- ko if: shouldShowIncompleteWorkflowsModal() -->
+    <div 
+        style="
+            position: absolute;
+            width: 100%;
+            height: 100%;
+            background: rgba(0, 0, 0, .5);
+            z-index: 1;
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            cursor: pointer;
+        "
+    >
+        <div 
+            style="background-color: white; width: 1000px; max-height: 480px; border-radius: 4px; cursor: default; overflow: hidden;"
+        >
+            <div style="width: 100%; display: flex; justify-content: flex-end; padding: 10px 12px;">
+                <i 
+                    class="fa fa-times" style="font-size: 20px; cursor: pointer;"
+                    data-bind="click: function(){ $data.shouldShowIncompleteWorkflowsModal(false); }"
+                ></i>
             </div>
-            <p data-bind="text: workflow.desc" class="workflow-select-desc"></p>
-        </div></a>
-        <!-- /ko -->
+            <div style="display: flex; flex-direction: column; align-items: center;">
+                <h2>{% trans 'You have incomplete workflows' %}</h2>
+                <p>
+                    {% trans 'Please either complete or delete the workflows listed below.' %}
+                </p>
+            </div>
+
+            <div style="padding: 40px 80px; display: flex; justify-content: center;">
+                <table class="table table-striped" style="border: 1px solid #ddd;">
+                    <thead>
+                        <tr style="display: flex; justify-content: space-between;">
+                            <th style="width: 219px;">
+                                {% trans 'Created' %}
+                            </th>
+                            <th style="width: 400px;">
+                                {% trans 'Workflow Name' %}
+                            </th>
+                            <th style="width: 219px;">
+                                {% trans 'Link' %}
+                            </th>
+                        </tr>
+                    </thead>
+                    <tbody style="display:block; overflow:auto; max-height:200px; width:100%;">
+                        <!-- ko foreach: $data.incompleteWorkflows -->
+                        <tr style="display: flex; justify-content: space-between;">
+                            <td style="width: 219px;" data-bind="text: $data.created"></td>
+                            <td style="width: 400px;" data-bind="text: $data.workflowname"></td>
+                            <td style="width: 219px;">
+                                <a 
+                                    data-bind="attr: {'href': `/plugins/${$data.workflowname}?workflow-id=${$data.workflowid}`}"
```

Comment:
I've noticed sometimes arches projects vary in where their core arches (or in this case, arches app) URLs are included, sometimes under /arches, etc. Do we have any javascript constants/utils for getting the actual route? If not, maybe let's draw up a follow-up issue.

Reply:
ya there's no set-in-stone convention, and the arches-core url for plugins accepts a UUID.

Reply:
ya there's not set-in-stone convention for this